docker_stats_exporter.js

```diff
@@ -160,12 +166,14 @@ async function gatherMetrics() {
             // Memory
             if (result['memory_stats']) {
                 let memUsage = result['memory_stats']['usage'];
+                let memUsageRss = result['memory_stats']['stats']['rss'];
```

Comment:
Thanks for the PR.

Can you check for existence of `result['memory_stats']['stats']` before setting the value please, because it may be present in all environments?

```suggestion
                let memUsageRss = result['memory_stats']['stats'] && result['memory_stats']['stats']['rss'] ? result['memory_stats']['stats']['rss'] : 0;
```